src/store/page-load/saga.test.ts

```diff
@@ -106,6 +106,33 @@ describe('page-load saga', () => {
     expect(storeState.pageload.isComplete).toBe(true);
     expect(history.replace).not.toHaveBeenCalledWith({ pathname: '/login' });
   });
+
+  it('redirects authenticated user from /reset-password to main page', async () => {
+    const initialState = { pageload: { isComplete: false } };
+
+    let history = new StubHistory('/reset-password');
+    const { storeState: resetPasswordStoreState } = await expectSaga(saga)
+      .withReducer(rootReducer, initialState as any)
+      .provide(stubResponses(history, true))
+      .run();
+
+    // redirected from /reset-password to /0.wilder.channels
+    expect(history.replace).toHaveBeenCalledWith({ pathname: '/0.wilder/channels' });
+    expect(resetPasswordStoreState.pageload.isComplete).toBe(true);
+  });
```

Comment:
I just saw this test in other PR..do you need to rebase or something?

Reply:
Updated this PR to point to the other PR so you now shouldn't see those tests. The branch for this PR was created from the other :)